README.md

```diff
@@ -113,7 +113,7 @@ bin/drain.sh
 
 ## Releasing
 
-[Releasing Guide](https://github.com/astronomerio/helm.astronomer.io/blob/master/RELEASING.md)
+[Releasing Guide](https://github.com/astronomer/astronomer/blob/master/RELEASING.md)
```

Comment:
i can't find this document....`RELEASING.md`
https://github.com/astronomer/helm.astronomer.io/blob/container_memory_at_the_limit/RELEASING.md
cc @schnie @sjmiller609